static_inverse_kinematics.py

```diff
@@ -60,7 +60,33 @@ def _marker_jac(self, q: np.ndarray, xp_markers: np.ndarray, n_frame: int):
 
         return jac
 
-    def solve(self, method: str = "lm"):
+    def optimize(self, n_frame: int, method: str, bounds: tuple() = None):
+        print(f" ****   Frame {n_frame}  ****")
```

Comment:
please add a docstring and modify the print to display
```python3
 print(f" ****   Frame {n_frame} / {self.nb_frame} ****")```

Reply:
done

Reply:
@AurelienRenou  This one seems not done

Reply:
yes, now it's done